mtp_bank_admin/apps/mtp_auth/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.conf.urls import url
+from django.core.urlresolvers import reverse_lazy
+
+from moj_auth import views
+
+urlpatterns = [
+    url(r'^login/$', views.login, {
+            'template_name': 'mtp_auth/login.html',
+        }, name='login'),
+    url(
+        r'^logout/$', views.logout, {
+            'template_name': 'mtp_auth/login.html',
+            'next_page': reverse_lazy('auth:login'),
+        }, name='logout'
+    ),
+]
```

Comment:
Can we move all the URL/view/template definitions for the auth pages into the auth app and make them overridable in consumers?